README.md

```diff
@@ -593,6 +593,7 @@ sections = {
     {
       'filename',
       file_status = true,      -- Displays file status (readonly status, modified status)
+      newfile_status = false   -- Whether show new file status, default disabled
```

Comment:
I think you should define what you mean by 'new file status'. Also default disabled isn't necessary as =false already states that

Reply:
Updated.